lib/google-apis/auth-client.js

```diff
@@ -57,6 +65,8 @@ function AuthClient(projectId, shouldReportErrors, key) {
 
   this._authClient = null;
   this._hasInited = false;
+  this._cannotInit = false;
+  this._initError = null;
```

Comment:
Could these two properties be elided? The non-nullness of `_initError` implies `_cannotInit` if I am not mistaken.